source/agora/test/Base.d

```diff
@@ -1964,8 +1964,10 @@ public struct TestConf
     /// Out of 100
     public ubyte validator_tx_fee_cut = 70;
 
-    /// How frequent the payments to Validators will be
-    public uint payout_period = 5;
+    /// How frequent the payments to Validators will be. If too low then tests
+    /// which assert all blocks are the same will sometimes fail if the catchup
+    /// is after height goes beyond a payment block.
+    public uint payout_period = 200;
```

Comment:
The default should be the `ConsensusParams`'s default, that is, 144 (`ConsensusParams.init.payout_period`)

Reply:
Updated.